tests/end_to_end/netsh_test_helper.h

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include <windows.h>
+#include <netsh.h> // Must be included after windows.h
+#include <string.h>
+#include "bpf.h"
+#include "capture_helper.hpp"
+#include "catch_wrapper.hpp"
+#include "elf.h"
+#include "links.h"
+#include "maps.h"
+#include "pins.h"
+#include "programs.h"
+
+#pragma region
+// Mock Netsh.exe APIs.
+
+DWORD WINAPI
+PreprocessCommand(
+    _In_opt_ HANDLE hModule,
+    _Inout_updates_(dwArgCount) LPWSTR* ppwcArguments,
+    _In_ DWORD dwCurrentIndex,
+    _In_ DWORD dwArgCount,
+    _Inout_updates_opt_(dwTagCount) TAG_TYPE* pttTags,
+    _In_ DWORD dwTagCount,
+    _In_ DWORD dwMinArgs,
+    _In_ DWORD dwMaxArgs,
+    _Out_writes_opt_(dwArgCount - dwCurrentIndex) DWORD* pdwTagType)
+{
+    UNREFERENCED_PARAMETER(hModule);
+
+    DWORD argc = dwArgCount - dwCurrentIndex;
```

Comment:
.h files should not include function definitions, it causes linker problems.  Revert.